cmd/syncthing/cli/utils.go

```diff
@@ -127,3 +130,20 @@ func prettyPrintResponse(c *cli.Context, response *http.Response) error {
 	// TODO: Check flag for pretty print format
 	return prettyPrintJSON(data)
 }
+
+func getDB() (backend.Backend, error) {
+	return backend.OpenLevelDBRO(locations.Get(locations.Database))
+}
+
+func nulString(bs []byte) string {
+	for i := range bs {
+		if bs[i] == 0 {
+			return string(bs[:i])
+		}
+	}
+	return string(bs)
+}
+
+func normalizePath(path string) string {
+	return filepath.ToSlash(filepath.Clean(path))
```

Comment:
YAGNI, but wonder if we should handle people giving us the full absolute path etc, and trimming the folder path prefix.

Reply:
I am not convinced we need to that. And if we do, then I'd argue that should happen on the api side of things.